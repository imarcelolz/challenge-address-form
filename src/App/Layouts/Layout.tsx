import logo from '@/Assets/Images/Logo.svg';
import { createMuiTheme, Grid, makeStyles, MuiThemeProvider, Paper, responsiveFontSizes } from '@material-ui/core';
import DescriptionIcon from '@material-ui/icons/Description';
import React from 'react';
import { styles } from './Layout.styles';

const useStyles = makeStyles(styles);
const theme = responsiveFontSizes(createMuiTheme(), { factor: 0.1 });

export const Layout = (props) => {

  const classes = useStyles();
  return (
    <MuiThemeProvider theme={theme}>
      <Grid>
        <Grid container direction="row" alignItems="center" component={Paper} justify="space-between" className={classes.Header}>
          <Grid item className={classes.Logo} xs={6}>
            <img alt={logo} src={logo} />
          </Grid>
          <Grid item>
            <a href="https://imarcelolz.github.io"><DescriptionIcon /></a>
          </Grid>
        </Grid>
        <Grid container item md={12} className={classes.Content}>
          {props.children}
        </Grid>
      </Grid>
    </MuiThemeProvider>
  );
}
