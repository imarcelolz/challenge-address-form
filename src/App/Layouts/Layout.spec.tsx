import { shallow } from 'enzyme';
import React from 'react';
import { Layout } from './Layout';

describe('Context: Rendering', () => {
  it('render without crash', () => {
    const wrapper = shallow(<Layout />);

    expect(wrapper).toMatchSnapshot();
  });
});
