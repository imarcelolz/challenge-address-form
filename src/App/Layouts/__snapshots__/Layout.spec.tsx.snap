// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Context: Rendering render without crash 1`] = `ShallowWrapper {}`;
